component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/x509Certificate/X509CertificateAuthenticator.java

```diff
@@ -209,6 +215,43 @@ protected void processAuthenticationResponse(HttpServletRequest httpServletReque
         }
     }
 
+    /**
+     * Resolves username of the subject attribute.
+     *
+     * @param identifier            username
+     * @param authenticationContext authentication context
+     *
+     * @return resolved username
+     */
+    private String resolveIdentifier(String identifier, AuthenticationContext authenticationContext)
```

Comment:
This method is to resolve the username for a given identifier right? In that case shall we rename this to resolveUsernameFromIdentifier to make this method more readable

Reply:
Resolved with 5f0e1d972a4fb2c2ddf7699da6e7935af4b44c31